news/1.breaking

```diff
@@ -0,0 +1,2 @@
+Drop python 2.7 support.
```

Comment:
This change log entry should also mention dropping of ZServer support.

Reply:
Oh, thanks, indeed, although it is already _broken_ / not being used if you are in python 3 anyway.